frontera/core/manager.py

```diff
@@ -116,14 +116,14 @@ def __init__(self, request_model, response_model, backend, logger, event_log_man
             self.start()
 
     @classmethod
-    def from_settings(cls, settings=None):
+    def from_settings(cls, settings=None, crawler_settings=None):
```

Comment:
Ok @sibiryakov, thank you for the feedback. I'm working on the changes you suggested.